stdlib/itertools.pyi

```diff
@@ -30,7 +31,9 @@ class count(Iterator[_N], Generic[_N]):
     @overload
     def __new__(cls) -> count[int]: ...
     @overload
-    def __new__(cls, start: _N, step: _NStep = ...) -> count[_N]: ...
+    def __new__(cls, start: _N, step: _Step = ...) -> count[_N]: ...
+    @overload
+    def __new__(cls, step: _N) -> count[_N]: ...
```

Comment:
```suggestion
    def __new__(cls, *, step: _N) -> count[_N]: ...
```

I don't think that makes a huge difference, but since `count(3)` already matches the previous overload, this makes the purpose of this overload a little clearer.